### PR TITLE
Version working with Prometheus 2.0

### PR DESCRIPTION
**Pull request by sebthemonster**
I worked to adapt your dockprom to work with Prometheus 2.0.
It's my first config on Prometheus/Grafana, please could you be indulgent with me. I accept any comments to be better.

Thanks a lot

### CODE REVIEW

File: grafana/dashboards/docker_host.json [hunk 1]
```diff
@@ -477,7 +477,7 @@
           },
           "targets": [
             {
-              "expr": "sum(node_filesystem_free{fstype=\"aufs\"})",
+              "expr": "sum(node_filesystem_free{fstype=\"btrfs\"})",
```
Comment (stefanprodan) [inline grafana/dashboards/docker_host.json]: Can you please change it back to aufs

File: grafana/dashboards/docker_containers.json [hunk 2]
```diff
@@ -843,7 +843,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(container_cpu_user_seconds_total{image!=\"\", container_label_org_label_schema_group!=\"monitoring\"}[1m]) * 100  / count_scalar(node_cpu{mode=\"user\"})) by (name)",
+              "expr": "sum by (name) (rate(container_cpu_usage_seconds_total{image!=\"\",container_label_org_label_schema_group=\"\"}[1m])) / scalar(count(node_cpu{mode=\"system\"})) * 100",
```
Comment (stefanprodan) [inline grafana/dashboards/docker_containers.json]: instead of `mode="system"` it should be `mode="user"`

Comment (sebthemonster) [reply grafana/dashboards/docker_containers.json]: why mode="user" is better than mode="system" ?

File: grafana/dashboards/docker_host.json (hunk 1 above)
Comment (sebthemonster) [reply grafana/dashboards/docker_host.json]: ok. I change it.

**Comment by sebthemonster** [comment]
I made requested changes in next commit.

File: grafana/dashboards/docker_containers.json (hunk 2 above)
Comment (stefanprodan) [reply grafana/dashboards/docker_containers.json]: I'm not saying is better, I just want have the same graphs like before.

**Comment by stefanprodan** [comment]
Thanks @sebthemonster

**Comment by pascalandy** [comment]
This is awesome @sebthemonster !

**Comment by stefanprodan** [comment]
@pascalandy can you please give it a try? be aware that the database is not backwards compatible, you need to start fresh.

**Comment by pascalandy** [comment]
With pleasure. Give me about 48h

**Comment by pascalandy** [comment]
I was about to try it but play-with-docker is down at the moment. I'll try it later on :-p

**Comment by uLan08** [comment]
Hi, could anyone confirm that it's working perfectly? I am experiencing problems with the update.

**Comment by sebthemonster** [comment]
Hi, i confirm that it's working. I installed it in production what i couldn't do it before.
I found a problem with "storage load" request that i fixed it directly in Grafana dashboard because i don't know if it's a problem due to my configuration or to the project.

The problem is that the initial request get multiple results : 

`(node_filesystem_size{fstype=\"aufs\"} - node_filesystem_free{fstype=\"aufs\"}) / node_filesystem_size{fstype=\"aufs\"}  * 100`

>[ 
> {device="none",fstype="aufs",instance="nodeexporter:9100",job="nodeexporter",mountpoint="/rootfs/var/lib/docker/aufs/mnt/278bdd5a9d024f397bb19ead082ac2d54253e794e4c0493d8f8f734ac0ac46a8"}
> {device="none",fstype="aufs",instance="nodeexporter:9100",job="nodeexporter",mountpoint="/rootfs/var/lib/docker/aufs/mnt/0f8915002f902baf89e82dcb30911775bac922cceaed131e33dca2b23e2706be"}
> {device="none",fstype="aufs",instance="nodeexporter:9100",job="nodeexporter",mountpoint="/rootfs/var/lib/docker/aufs/mnt/0652aec3a6ef6531e133e707b7eb382841399d2d41ac20573a841c27b6326191"}
> {device="none",fstype="aufs",instance="nodeexporter:9100",job="nodeexporter",mountpoint="/"}
> [...]
> ]

To fix it i set request as this : 
`(node_filesystem_size{fstype=\"aufs\",mountpoint=\"/\"} - node_filesystem_free{fstype=\"aufs\",mountpoint=\"/\"}) / node_filesystem_size{fstype="aufs",mountpoint=\"/\"}  * 100`

Maybe this is not the right solution ? Can you say me if you're encountered same problem ?

**Pull request merged**